main/lsp/ErrorReporter.cc

```diff
@@ -139,10 +172,19 @@ u4 ErrorReporter::lastDiagnosticEpochForFile(core::FileRef file) {
     return getFileErrorStatus(file).lastReportedEpoch;
 }
 
-ErrorEpoch::ErrorEpoch(ErrorReporter &errorReporter, u4 epoch,
+void ErrorReporter::sanityCheck() const {
+    DEBUG_ONLY(u4 errorCount = 0; for (auto &status
+                                       : this->fileErrorStatuses) {
+        if (status.lastReportedEpoch >= this->lastFullTypecheckEpoch) {
+            errorCount += status.errorCount;
+        }
+    } ENFORCE(errorCount == this->clientErrorCount));
```

Comment:
clang-format really garbled this. also, it looks like we usually just do

```cpp
    if (!debug_mode) {
        return;
    }
```

at the top of sanityCheck methods. Do you want to reformat use that instead of `DEBUG_ONLY`?

Reply:
In the past I was encouraged to use `DEBUG_ONLY` for readability. I'm happy to use `if (!debug_mode)`.